src/App.tsx

```diff
@@ -15,36 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  byAlphabet = 'byAlphabet',
+  byLength = 'byLength',
+}
+
+type SortConfig = {
+  type: string | null;
+  reversed: boolean;
+};
+
 export const App: React.FC = () => {
+  const [sortConfig, setSortConfig] = useState<SortConfig>({
+    type: '',
+    reversed: false,
+  });
+
+  const handleSort = (goods: string[], config: SortConfig) => {
```

Comment:
The function's name 'handleSort' is not starting with a verb that indicates what the method does. Consider renaming it to more accurately reflect its action, such as 'getSortedGoods' or 'sortGoods'.